### ISSUE TITLE
Update 3.x doc to reflect that whole var dir is writable

### ISSUE THREAD

**Issue by andrerom** [symfony/symfony-docs#6132]
As of 3.0, writeable files are placed in `var` folder, however the doc for directory permissions does not reflect this other then just mapping the old doc from `app => var`.

If this is changed to give write access to `var` it would:
- solve var/sessions write issue introduced in https://github.com/symfony/symfony-standard/commit/dcf4eadc767246546729813d88a3d1592e954363
- Make it easier for systems like eZ to add own folder for user uploaded binary files ala asked for in #6095 without having to document this specifically for dir permissions *(however a convention on this would be nice, so I'm not saying solving this will fully solve it)*
- ..

In summary, changing:
```bash
$ sudo chmod +a "$HTTPDUSER allow delete,write,append,file_inherit,directory_inherit" var/cache var/logs
```
to
```bash
$ sudo chmod +a "$HTTPDUSER allow delete,write,append,file_inherit,directory_inherit" var
```
 *( same goes for all other commands dealing with directory permissions [in doc](http://symfony.com/doc/current/cookbook/configuration/configuration_organization.html) )*

**Comment by andrerom** [comment]
PR: #6136
